store/core/cache.go

```diff
@@ -0,0 +1,82 @@
+package corestore
+
+import (
+	"sync"
+	"time"
+)
+
+type item struct {
+	key      string
+	value    string
+	expireAt int64
+}
+
+func (i *item) expired() bool {
+	if i.expireAt == 0 {
+		return false
+	}
+	return time.Now().Unix() >= i.expireAt
+}
+
+// Cache is ... a cache
+// it will cleanup the expired items every interval time
+type Cache struct {
+	interval time.Duration
+	items    *sync.Map
+}
+
+func (c *Cache) cleanup() {
+	ticker := time.NewTicker(c.interval)
+	for range ticker.C {
```

Comment:
im not quite sure shall we need to close the ticker here, but the rest of all LGTM

Reply:
`client` will always be there so I think no need to close `ticker.C`, this comes with `client`